config/http_config.go

```diff
@@ -489,8 +494,10 @@ func (rt *oauth2RoundTripper) RoundTrip(req *http.Request) (*http.Response, erro
 		if err != nil {
```

Comment:
Perhaps should we lock over here instead so we don't get multiple threads reading the file at once and assessing the same thing?

Reply:
Just above that line, once again GitHub not letting me comment

Reply:
Actually wait never mind that's pointless.